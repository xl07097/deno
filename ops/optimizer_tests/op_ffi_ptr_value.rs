pub fn op_ffi_ptr_value(ptr: *mut c_void, out: &mut [u32]) {
  // ...
}
