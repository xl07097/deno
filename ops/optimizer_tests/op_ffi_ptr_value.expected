=== Optimizer Dump ===
returns_result: false
has_ref_opstate: false
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: true
fast_result: Some(Void)
fast_parameters: [V8Value, Pointer, Uint32Array]
transforms: {0: Transform { kind: PtrVoid, index: 0 }, 1: Transform { kind: SliceU32(true), index: 1 }}
is_async: false
fast_compatible: true
